.github/workflows/pull-requests.yaml

```diff
@@ -11,6 +11,21 @@ on:
 
 jobs:
 
+  set-build-properties:
```

Comment:
Shall we remove this from here as the output isn't passed into pr-obr.yaml as an input so nothing is done with the galasa-version here